### PR TITLE
DOT-7404: COD-Theme > Collection page integration

### PR DESCRIPTION
**Pull request by adildev101**
## JIRA Ticket

[DOT-7404](https://nextmediama.atlassian.net/browse/DOT-7404)

## QA Steps

-  [ ] `pnpm i`
-  [ ] `pnpm dev`

## Note

Leave empty when you have nothing to say

[DOT-7404]: https://youcanshop.atlassian.net/browse/DOT-7404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: sections/main-list-collections.liquid [hunk 1]
```diff
@@ -1,21 +1,15 @@
 {{ 'queries-handler.css' | asset_url | stylesheet_tag }}
-{{ 'featured-products.css' | asset_url | stylesheet_tag }}
+{{ 'collection-listing.css' | asset_url | stylesheet_tag }}
 
 {%- assign per_page = section.settings.items_per_page -%}
-<div class='yc-featured-products container'>
```
Comment (bj-anas) [inline sections/main-list-collections.liquid:5]: Screen Shot 2023-02-21 at 17 24 30

you need to remove title settings also

File: snippets/breadcrumbs.liquid [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="breadcrumbs-container">
+  <div class="breadcrumbs-listings">
+    <div class="listings__item">
+      <a href="/" class="first-listing">الرئيسية</a>
```
Comment (bj-anas) [inline snippets/breadcrumbs.liquid:4]: this content should be translated

File: snippets/breadcrumbs.liquid [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    </div>
+    <div class="listings__item">
+      <img src='{{ 'left-arrow.png' | asset_url }}'>
+    </div>
+    <div class="listings__item">
+      <a href="/collections" class="second-listing">التصنيفات</a>
```
Comment (bj-anas) [inline snippets/breadcrumbs.liquid:10]: this should not be static it should be getting the category name

File: snippets/breadcrumbs.liquid [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    </div>
+  </div>
+
+  <div class="breadcrumbs-name">
+    <h1>التصنيفات</h1>
```
Comment (bj-anas) [inline snippets/breadcrumbs.liquid:15]: same here you should use the category name

File: snippets/breadcrumbs.liquid (hunk 3 above)
Comment (adildev101) [reply snippets/breadcrumbs.liquid:10]: we are not inside a category we are showing all the collections

Comment (bj-anas) [reply snippets/breadcrumbs.liquid:10]: ok but should be translated too

**Review by bj-anas — APPROVED** [review]
LGTM

**Pull request merged**